.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,41 @@
+---
+name: Bug report
+about: Create a report to help us improve
+title: ''
+labels: requires triage
+assignees: ''
+
+---
+
+**Describe the bug**
+<!-- A clear and concise description of what the bug is. -->
+
+**Expected behavior**
+<!-- A clear and concise description of what you expected to happen. -->
+
+**To Reproduce**
+Please provide a [Minimal, Complete, and Verifiable](http://stackoverflow.com/help/mcve) example. 
+See also [Reporting Bugs](https://www.sqlalchemy.org/participate.html#bugs) on the website.
```

Comment:
alembic is not as easy here but yes, we should encourage them to try :)

Reply:
I'll try to clarify a bit, to suggest to at provide the migration that causes problems and the sqlalchemy table/model